source/openapi-admin-v3.yaml

```
@@ -7109,6 +7109,19 @@ components:
                     can process events in parallel. If `false`, event
                     ordering is enabled and the Trigger executes events
                     serially.
+            error_handler:
+              type: object
+              description: An object that defines custom error handling for an AWS EventBridge trigger. Valid only for `AWS_EVENTBRIDGE` event processors. 
+              properties:
+                config:
+                  type: object
+                  properties:
+                    enabled:
+                      type: boolean
+                      description: If 'true', custom error handling is enabled for the AWS EventBridge trigger. 
+                    function_id:
+                      type: string
+                      description: The ID of the custom error handler invoked when the AWS EventBridge trigger fails and cannot be successfully retried.
```

Just a thought -- custom error handlers are just functions at the end of the day. Would it be more clear to reword this to something closer to this:
```suggestion
                      description: The ID of the function invoked when the AWS EventBridge trigger fails and cannot be successfully retried.
```
If it isn't confusing due to the structure of the object or if we are trying to communicate something by using `custom error handler` then I am on board :) My suggestion is to hopefully alleviate any confusion that they need to look for custom error handler in the UI/app config rather than just any function they have created.

---

No, but I was trying to be consistent with the naming. Agree it's potentially confusing here.